compositor_web/src/wasm/input_uploader.rs

```diff
@@ -20,7 +20,8 @@ impl InputUploader {
         let pts = Duration::from_millis(input.pts_ms as u64);
         let mut frames = HashMap::new();
         for frame in input.frames.entries() {
-            let types::InputFrame { id, frame } = frame?.try_into()?;
+            // TODO: MP4 are not calculated correctly
+            let types::InputFrame { id, frame, .. } = frame?.try_into()?;
```

Comment:
the fix is out of scope for this PR, but decided to leave the frame set type change